src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -33,7 +33,7 @@ const AppLauncher = _.flow(
     h(RuntimeStatusMonitor, {
       runtime,
       onRuntimeStartedRunning: () => {
-        Ajax().Metrics.captureEvent(Events.applicationLaunch, { app })
+        Ajax().Metrics.captureEvent(Events.applicationLaunch, { application })
```

Comment:
We've already been collecting metrics with a particular shape, and this event is used for Galaxy as well, so we shouldn't change the shape of the data here, or it will make analytics harder.